api/models/Event.js

```diff
@@ -9,11 +9,13 @@ module.exports = {
 
   attributes: {
     user: "string",
```

Comment:
Gagh, I hate the fact that I sorted everything using userid at the start. I really want to go through everything and change it so that we ignore the user id completely, and only ever use the username. It shouldn't actually be too hard.